x-pack/plugins/security_solution/public/timelines/components/open_timeline/helpers.ts

```diff
@@ -407,7 +408,7 @@ export const dispatchUpdateTimeline = (dispatch: Dispatch): DispatchUpdateTimeli
     dispatch(dispatchAddGlobalTimelineNote({ noteId: newNote.id, id }));
   }
 
-  if (!duplicate) {
+  if (!duplicate || forceNotes) {
```

Comment:
I'm wondering if we could remove `forceNotes` and just put this line as `if (!duplicate && notes != null)`

Reply:
Otherwise I've tested both cases for missing filed in dataProvider, and investigate timeline in alerts table, all looks good.

Reply:
I do not think so because we only to push notes for templates, because if you duplicate a timeline with notes, you do not want the notes when you duplicate it.